test/build-localnet-validator.sh

```diff
@@ -1,6 +1,29 @@
 #! /bin/bash
 
-echo "Make sure the validator account are imported"
+# The faucet address
+FAUCET_ADDRESS="one1zksj3evekayy90xt4psrz8h6j2v3hla4qwz4ur"
+
+# Get list of the current keys
+KEYS_OUTPUT=$(hmy keys list)
+
+# Check if the hmy executed successfully
+if [ $? -ne 0 ]; then
+  echo "Error: Failed to execute 'hmy keys list'. Please ensure the hmy tool is installed and configured properly."
+  exit 1
+fi
+
+# Search for the faucet address in the output
+# if the faucet account doesn't exist, show error 
+if echo "$KEYS_OUTPUT" | grep -q "$FAUCET_ADDRESS"; then
+  echo "The faucet account is set"
+else
+  echo "validator creation failed"
+  echo "the faucet address '$FAUCET_ADDRESS' is not found"
+  echo "the keys location is: $(hmy keys location)"
+  echo "Make sure the validator account are imported"
```

Comment:
replace validator with faucet

Reply:
the validator account are imported right after that

Reply:
done